lib/Server.js

```diff
@@ -668,6 +674,10 @@ class Server {
     }, this);
   }
 
+  getStats(statsObj) {
+    return statsObj.toJson(this.stats);
+  }
```

Comment:
Let's move
```
this.stats =
      options.stats && Object.keys(options.stats).length
        ? options.stats
        : Server.DEFAULT_STATS;
```

logic here, we don't need new property like `stats`

Reply:
Oh, good point, will update.

Reply:
I still need to keep the property to have access to `options.stats`.

Reply:
Not sure it would be useful to move the logic since I still need to keep that `this.stats` so I can use `options.stats` in `getStats()`.
What do you think @evilebottnawi ?

Reply:
Yep, you are right, we should rewrite this in next major release